WordPress/Classes/Services/BlogJetpackSettingsService.swift

```diff
@@ -79,51 +79,51 @@ struct BlogJetpackSettingsService {
             return
         }
         guard let remoteAPI = blog.wordPressComRestApi(),
-            let blogDotComId = blog.dotComID as? Int,
-            let blogSettings = blog.settings else {
-                failure(nil)
-                return
+            let blogDotComId = blog.dotComID as? Int else {
+            failure(nil)
```

Comment:
This seems like a more correct approach vs line 21?

Or at least...odd that they differ?

Reply:
I'm going to mark this conversation as resolved because the approach at line 21 it refers to has been updated to match this one. 

See also https://github.com/wordpress-mobile/WordPress-iOS/pull/19920#discussion_r1072944273

I'm marking it as resolved because otherwise the comment would read quite confusing, since the code here and at line 21 is now the same.